packages/components/psammead-embed-error/src/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { string, bool } from 'prop-types';
+import styled from 'styled-components';
+import { C_CHALK, C_EBON } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_BODY_COPY } from '@bbc/gel-foundations/typography';
+import { BBC_BLOCKS } from '@bbc/psammead-assets/svgs';
+
+const StyledEmbedError = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_BODY_COPY};
+  background-color: ${C_CHALK};
+  background-image: url(data:image/svg+xml;base64,${BBC_BLOCKS});
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 38.2%;
+  color: ${C_EBON};
+  display: flex;
+  flex-direction: column;
+  height: ${({ fillViewport }) => (fillViewport ? '100vh' : '100%')};
+  justify-content: flex-end;
+`;
+
+const StyledErrorMessage = styled.p`
```

Comment:
I think we could just use `<strong>` directly instead of wrapping it in a <p>, @simonsinclair what do you think ?

Reply:
Technically we could, yes. Normally I'd mark up a paragraph of text in a `<p>`, but I guess we can't always be sure we're going to receive a paragraph here. Good spot. I'll amend this now.